python/ray/tests/test_autoscaler.py

```diff
@@ -2313,17 +2324,25 @@ def do_nothing(*args, **kwargs):
             update_interval_s=0,
         )
         autoscaler.update()
+        time.sleep(3)
         autoscaler.update()
+        time.sleep(3)
         self.waitForNodes(2)
+        time.sleep(3)
         self.provider.finish_starting_nodes()
+        time.sleep(3)
         autoscaler.update()
+        time.sleep(3)
         self.waitForNodes(2, tag_filters={TAG_RAY_NODE_STATUS: STATUS_UP_TO_DATE})
         runner.calls = []
         new_config = copy.deepcopy(SMALL_CLUSTER)
         new_config["worker_setup_commands"] = ["cmdX", "cmdY"]
         self.write_config(new_config)
+        time.sleep(3)
         autoscaler.update()
+        time.sleep(3)
```

Comment:
Do we need that many sleeps?

Also could you add a comment saying that these sleeps will be removed after autoscaler v2